oci/config/convert/runtime.go

```diff
@@ -148,11 +147,11 @@ func MutateRuntimeSpec(g rgen.Generator, rootfs string, image ispec.Image) error
 	}
 
 	// Set optional fields
-	var ports []string
-	for port := range image.Config.ExposedPorts {
-		ports = append(ports, port)
+	ug, err := ugen.NewFromImage(image)
```

Comment:
If we're going to do this, please use `ug` everywhere in this function where we reference `image`directly.